.template/addons/svgeez/.github/wiki/template.rb

```diff
@@ -0,0 +1,16 @@
+
+# frozen_string_literal: true
+
+if Dir.exist?('.github/wiki')
+  use_source_path __dir__
+
+  copy_file 'Add-new-svg-icon.md', '.github/wiki/Add-new-svg-icon.md'
+
+  # SVG Sprite
+  insert_into_file '.github/wiki/_Sidebar.md', after: /## Operations.*\n/ do
+    <<~RUBY
+
+      - [[Add new svg icon]]
```

Comment:
What do you think of renaming it to `Managing SVG icons` as the wiki is about managing the icons?

Reply:
Yeah. This title is more relevant 👍🏼